tests/testthat/test-scale-expansion.R

```diff
@@ -116,6 +116,6 @@ test_that("expand_limits_scale_discrete() begrudgingly handles numeric limits",
       coord_limits = c(NA, NA),
       range_continuous = c(-15, -2)
     ),
-    c(-15, -2)
+    c(-16, -1)
```

Comment:
I didn't understand this test, even looking through #3918 and #3919 where this was introduced.
Why should scale expansion take the narrower range here?
In any case, this PR essentially takes the `range()` of both ranges.